AttoBot/lambda_function.py

```diff
@@ -25,6 +25,12 @@
 
 SECRET = os.environ["SECRET"]
 
+TAG_REQ = "\n".join((
+    "Please make sure that:",
+    "- [ ] CI passes for supported Julia versions (if applicable).",
+    "- [ ] Version bounds are up to date."
```

Comment:
Yeah, this won't be totally clear to a newbie. Do you have anything we can link to?